src/commands/ping.ts

```diff
@@ -12,7 +12,7 @@ export const ping: CommandInt = {
       .setDescription("One moment please.");
     const pingMessage = await message.channel.send(msg);
     const pingTime = Math.round(
-      pingMessage.createdTimestamp - message.createdTimestamp
+      pingMessage.createdTimestamp - message.createdTimestamp - 3000;
```

Comment:
```suggestion
      pingMessage.createdTimestamp - message.createdTimestamp - 3000
```

Removing the semi-colon here should fix the linting error.